Getting-Started-With-Orleans/Grain-Persistence.md

```diff
@@ -297,4 +325,35 @@ Any other failure conditions from a write operation _should_ cause the write `Ta
 
 Individual storage providers should decide how best to store grain state – blob (various formats / serialized forms) or column-per-field are obvious choices.
 
-The basic storage provider for Azure Table encodes state data fields into a single table column using Orleans binary serialization.
\ No newline at end of file
+The basic storage provider for Azure Table encodes state data fields into a single table column using Orleans binary serialization.
+
+
+## ADO.NET Persistence Rationale <a name="ADONETPersistenceRationale"></a>
```

Comment:
this should be moved to the "relational storage" page. Also, there's some duplication between this and "relational storage"